README.md

```diff
@@ -914,6 +914,9 @@ Terser.minify(code, { mangle: { toplevel: true } }).code;
   Pass an empty string `""` to enable, or a non-empty string to set the debug suffix.
 
 - `keep_quoted` (default: `false`) -— Only mangle unquoted property names.
+  - `true` -- quoted property names are reserved and any unquoted properties with the
+    same property name will not be mangled.
+  - `"strict"` -- advanced, all unquoted property names are mangled unless reserved.
```

Comment:
Can you add this explanation to the CLI part of the documentation too?

Reply:
Ah yes of course. Fixed. It seems the CLI-documentation is mostly less verbose so I chose a shorter explanation to avoid it taking up too much space.